.env.production.sample

```diff
@@ -90,3 +90,8 @@ SMTP_FROM_ADDRESS=notifications@example.com
 # Cluster number setting for streaming API server.
 # If you comment out following line, cluster number will be `numOfCpuCores - 1`.
 STREAMING_CLUSTER_NUM=1
+
+# Piwik ( Optional )
+# PIWIK_ENABLED=true
```

Comment:
That is correct. I wanted to adopt the same style as above for S3, and it was also useful not to put the piwik domain in the initial_state json, but I have no feelings toward it ;)

I'll change it tomorrow, need to go soon now :/